src/standaloneModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { type ModalComponent, type ModalName, manager } from './manager'
+
+
+// to render a modal in-place without registration required
+const standaloneModal = (modalName: ModalName, Component: ModalComponent) => {
+  // ATTN because we can render several instances of standaloneModal,
+  //  we should track them and render only in the active one
+  let lastId = 1
+  let activeIds: number[] = []
+
+  const Wrapper: React.FC = () => {
+    const [ , forceUpdate ] = useState(0)
+    const instanceIdRef = useRef<number>(0)
+
+    if (!instanceIdRef.current) {
+      instanceIdRef.current = lastId++
+    }
```

Comment:
why do we need this condition?
why can't we `const instanceIdRef = useRef<number>(1)` and without condition?

Reply:
it's an closure, each new modal will have own `instanceIdRef` value, not `1`
I thought it's more readable, but yes, we can remove the condition:
https://github.com/Local-Modules/modal/pull/1/commits/c90c53fd17c732b19c20cddb49762600c057a2d0

Reply:
oh, got it, thanks